added release notes up to 10.15.327.0 (#83)
SoftwareAG/c8y-releasenotes: content/release-10-16-0/applications-10-16-0-bundle/10-16-0-0.md

</tr>
</thead><tbody>

<tr>
<td>Cockpit</td>
<td>A widget developer can now set the global time context for the whole dashboard or a particular widget. It is also possible to share a dashboard with a specific time context via a URL with the time context parameters applied.</td>
<td>MTM-48051</td>
</tr>

<tr>
<td>Cockpit</td>

> <td>A new generic way of showing alerts in context of specific dashboard widgets has been implemented. For now it is only available in the "Pie chart" widget (users get an alert with a warning about negative values), but the solution is reusable and may be implemented in any widget.</td>

Reviewer: while not a major issue, i'm not sure about the parentheses here.

Author: What exactly do you mean?

Reviewer: i'd avoid them and make it a proper subordinate clause

> <td>A new generic way of showing alerts in context of specific dashboard widgets has been implemented. For now it is only available in the "Pie chart" widget (users get an alert with a warning about negative values), but the solution is reusable and may be implemented in any widget.</td>

Author: ```suggestion
<td>A new generic way of showing alerts in context of specific dashboard widgets has been implemented. For now it is only available in the "Pie chart" widget where users get an alert with a warning about negative values, but the solution is reusable and may be implemented in any widget.</td>

```

<td>MTM-47066</td>
</tr>

<tr>
<td>Cockpit</td>

> <td>Changes made to an item in the data point library will now also apply to already existing widgets if the item has been used in their configuration. For this purpose a new way of loading data for widgets called "DynamicDetailsResolver" has been added to the Web SDK (more details can be found in the Tutorial application).

Author: @janhommes Please see the comment above. Comes from @rssag , I added as again as it seems I closed it.

> <td>Changes made to an item in the data point library will now also be reflected on already existing widgets if this item from the data point library has been used in their configuration. For this purpose a new way of loading data for widgets called "DynamicDetailsResolver" has been added to the Web SDK (more details can be found in the Tutorial application).

Reviewer: ```suggestion
<td>Changes made to an item in the data point library will now also apply to already existing widgets if the item has been used in their configuration. For this purpose a new way of loading data for widgets called "DynamicDetailsResolver" has been added to the Web SDK (more details can be found in the Tutorial application).
```

also i think it's not sufficiently precise to say "a new way of loading data [...] called "DnyamicDetailsResolver" - since the way to load data is not the same as what i assume is a new class or object.
I would change it to "a new way of loading data for widgets has been added to the Web SDK with the class "DynamicDetails Resolver". The "class" term should be doublechecked with the developers though.

Author: @janhommes Please see Richards comment above under the suggestion.

</tr>
</thead><tbody>

<tr>
<td>Cockpit</td>
<td>A widget developer can now set the global time context for the whole dashboard or a particular widget. It is also possible to share a dashboard with a specific time context via a URL with the time context parameters applied.</td>
<td>MTM-48051</td>
</tr>

<tr>
<td>Cockpit</td>
<td>A new generic way of showing alerts in context of specific dashboard widgets has been implemented. For now it is only available in the "Pie chart" widget where users get an alert with a warning about negative values, but the solution is reusable and may be implemented in any widget.</td>

<td>MTM-47066</td>
</tr>

<tr>
<td>Cockpit</td>

> <td>Changes made to an item in the data point library will now also apply to already existing widgets if the item has been used in their configuration. For this purpose a new way of loading data for widgets called "DynamicDetailsResolver" has been added to the Web SDK (more details can be found in the Tutorial application).

Author: ```suggestion
<td>Changes made to an item in the data point library will now also apply to already existing widgets if the item has been used in their configuration. For this purpose a new way of loading data for widgets with the interface "DynamicDetailsResolver" has been added to the Web SDK . More details can be found in the Tutorial application.

```
